repository/expression.js

```diff
@@ -45,10 +45,19 @@ class ExpressionRepository {
     });
   }
 
-  deleteExpressionFromUsers(expressionId) {
+  deleteExpressionFromUser(expressionId, userId) {
     return UserExpression.destroy({
       where: {
         expressionId,
+        userId,
+      },
+    });
+  }
+
+  deleteUserFromExpression(userId) {
```

Comment:
There names are quite confusing. You are not actually deleting and user, you are just deleting the link between them. I think `deleteUserExpressionByUserId` is a better name.